lib/stytch/b2b_idp.rb

```diff
@@ -100,7 +100,7 @@ def introspect_token_network(
       }
       data['client_secret'] = client_secret unless client_secret.nil?
 
-      url = "/v1/public/#{@project_id}/oauth2/introspect"
+      url = @connection.url_prefix + "/v1/public/#{@project_id}/oauth2/introspect"
```

Comment:
While we _could_ continue to use the project scoped endpoint through the CNAME, let's try to use the pretty URLs where we can
```suggestion
      url = @connection.url_prefix + "/v1/oauth2/introspect"
```